stdlib/encodings/ascii.pyi

```diff
@@ -0,0 +1,30 @@
+import codecs
+from _typeshed import ReadableBuffer
+
+class Codec(codecs.Codec):
+    # At runtime, this is codecs.ascii_encode
+    @staticmethod
+    def encode(str: str, errors: str | None = None, /) -> tuple[bytes, int]: ...
+    # At runtime, this is codecs.ascii_decode
+    @staticmethod
+    def decode(data: ReadableBuffer, errors: str | None = None, /) -> tuple[str, int]: ...
+
+class IncrementalEncoder(codecs.IncrementalEncoder):
+    def encode(self, input: str, final: bool = False) -> bytes: ...
+
+class IncrementalDecoder(codecs.IncrementalDecoder):
+    def decode(self, input: ReadableBuffer, final: bool = False) -> str: ...
+
+class StreamWriter(Codec, codecs.StreamWriter): ...
+class StreamReader(Codec, codecs.StreamReader): ...
+
+# Note: encode being a decode function and decode being an encode function is accurate to runtime.
```

Comment:
Indeed, is that a bug in the runtime?

Reply:
Not sure. `StreamConverter` only exists on ascii and latin_1 encodings. Both do this, and there's nothing in the stdlib that makes use of them, so I don't know how they're intended to be used. I wondered the same thing, but I can't rule out that this is intended.

Reply:
They've also been this way for almost 25 years now.

It's possible that no one has ever used these classes for anything.